config.json

```diff
@@ -1,820 +1,818 @@
 {
-  "language": "Rust",
```

Comment:
This seems too important to shunt to the bottom of the file. I have evidence in https://github.com/exercism/bash/commit/6fabd6f26c76e799335ee43434826f9fac4ea3a9 and https://github.com/exercism/bash/pull/91 that this caused a mistaken assumption that the `language` attribute was absent. If this should indeed move, I need to hear a way to prevent others from making the same mistake.